ssp Dockerfile fails to build

Steps to reproduce:
```
cd spp
docker build .
```
Error after several minutes:
```
g++ -I/usr/share/R/include -DNDEBUG     -I./ -D_FASTMAP -DMAQ_LONGREADS	  -fpic  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -g  -c bed2vector.cpp -o bed2vector.o
In file included from /usr/include/c++/6/ext/hash_map:60:0,
                 from pc.h:4,
                 from bed2vector.cpp:1:
/usr/include/c++/6/backward/backward_warning.h:32:2: warning: #warning This file includes 
at least one deprecated or antiquated header which may be removed without further notice at 
a future date. Please use a non-deprecated interface with equivalent functionality instead. For
 a listing of replacement headers and interfaces, consult the file backward_warning.h. To 
disable this warning use -Wno-deprecated. [-Wcpp]
 #warning \
  ^~~~~~~
bed2vector.cpp: In function ‘SEXPREC* read_bed_ends(SEXP)’:
bed2vector.cpp:118:55: error: template argument 3 is invalid
   hash_map<string, int, hash<string>,equal_to<string> > cind_map;
                                                       ^
```

Failing on this line: 
https://github.com/Duke-GCB/GCB-Dockerfiles/blob/master/spp/1.13/Dockerfile#L38

--- a/spp/1.13/Dockerfile
+++ b/spp/1.13/Dockerfile
@@ -3,7 +3,7 @@ FROM r-base:3.2.4
 RUN apt-get update && apt-get install -y \
   libboost-all-dev \
   curl \
-  build-essential \
+  build-essential=12.2 \
   libncurses-dev \
   zlib1g-dev \
   gawk

